mountpoint-s3-client/src/s3_crt_client.rs

```diff
@@ -90,19 +90,24 @@ pub struct S3ClientConfig {
     request_payer: Option<String>,
     bucket_owner: Option<String>,
     max_attempts: Option<NonZeroUsize>,
+    read_backpressure: bool,
+    initial_read_window: usize,
 }
 
 impl Default for S3ClientConfig {
     fn default() -> Self {
+        const DEFAULT_PART_SIZE: usize = 8 * 1024 * 1024;
         Self {
             auth_config: Default::default(),
             throughput_target_gbps: 10.0,
-            part_size: 8 * 1024 * 1024,
+            part_size: DEFAULT_PART_SIZE,
             endpoint_config: EndpointConfig::new("us-east-1"),
             user_agent: None,
             request_payer: None,
             bucket_owner: None,
             max_attempts: None,
+            read_backpressure: false,
+            initial_read_window: DEFAULT_PART_SIZE,
```

Comment:
This is only first window size though, the only thing we really need is that this value should be more than 0 otherwise the request will be blocked from the start. If users want parallelisation they can always call `increment_read_window` with any size they want.